www/src/components/docs-table-of-contents.js

```diff
@@ -35,6 +35,10 @@ function createItems(items, location) {
               },
             },
           }}
+          activeStyle={{
```

Comment:
Looks like this never gets applied because Link from reach doesn’t take hash into account when evaluating whether the current link is active 🤦🏻‍♂️

We could use getProps to check ourselves and set these styles